tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -125,113 +227,319 @@ class InternalCallMapHandlerTest extends TestCase
+        'locale::lookup',
+        'logicexception::__construct',
         'long2ip',
-        'lzf_compress',
-        'lzf_decompress',
         'mail',
-        'mailparse_msg_extract_part',
-        'mailparse_msg_extract_part_file',
-        'mailparse_msg_extract_whole_part_file',
-        'mailparse_msg_free',
-        'mailparse_msg_get_part',
-        'mailparse_msg_get_part_data',
-        'mailparse_msg_get_structure',
-        'mailparse_msg_parse',
-        'mailparse_stream_encode',
-        'memcache_add',
-        'memcache_add_server',
-        'memcache_append',
-        'memcache_cas',
-        'memcache_close',
-        'memcache_connect',
-        'memcache_decrement',
-        'memcache_delete',
-        'memcache_flush',
-        'memcache_get_extended_stats',
-        'memcache_get_server_status',
-        'memcache_get_stats',
-        'memcache_get_version',
-        'memcache_increment',
-        'memcache_pconnect',
-        'memcache_prepend',
-        'memcache_replace',
-        'memcache_set',
-        'memcache_set_compress_threshold',
-        'memcache_set_failure_callback',
-        'memcache_set_server_params',
-        'mongodb\bson\tophp',
+        'messageformatter::format',
+        'messageformatter::formatmessage',
+        'messageformatter::parse',
+        'messageformatter::parsemessage',
+        'mongodb\bson\javascript::__construct',
+        'mongodb\driver\bulkwrite::delete',
+        'mongodb\driver\bulkwrite::update',
+        'mongodb\driver\clientencryption::__construct',
+        'mongodb\driver\clientencryption::decrypt',
+        'mongodb\driver\command::__construct',
+        'mongodb\driver\exception\runtimeexception::haserrorlabel',
+        'mongodb\driver\manager::__construct',
+        'mongodb\driver\manager::executebulkwrite',
+        'mongodb\driver\manager::executequery',
+        'mongodb\driver\manager::executereadcommand',
+        'mongodb\driver\manager::executewritecommand',
+        'mongodb\driver\query::__construct',
+        'mongodb\driver\server::executebulkwrite',
+        'mongodb\driver\server::executequery',
+        'mongodb\driver\server::executereadcommand',
+        'mongodb\driver\server::executereadwritecommand',
+        'mongodb\driver\server::executewritecommand',
+        'mongodb\driver\session::advanceoperationtime',
+        'mongodb\driver\topologydescription::hasreadableserver',
         'msg_receive',
         'msg_remove_queue',
         'msg_send',
         'msg_set_queue',
         'msg_stat_queue',
+        'multipleiterator::attachiterator',
+        'mysqli::poll',
         'mysqli_poll',
         'mysqli_real_connect',
+        'mysqli_stmt::__construct',
+        'mysqli_stmt::bind_param',
         'mysqli_stmt_bind_param',
+        'normalizer::getrawdecomposition',
+        'normalizer::isnormalized',
+        'normalizer::normalize',
         'normalizer_get_raw_decomposition',
-        'oauth_get_sbs',
```

Comment:
These should be re-added. I will work on an updated list.

Reply:
do you want to do that now or in a next PR?

Reply:
Let me check this list before merging.

Reply:
The arrays should show only additions now.

We have problems where certain functions aren't ignored for specific PHP versions. If you run the test with PHP 8.1 it will tell you to remove some functions from the ignore list even though they will fail with PHP 8.0. This can be addressed separately.

Reply:
Yeah, that's an issue, CI is failing because of that. Maybe we should build the list with the version psalm uses for tests now and enforce that version in tests? The day where we want to change it, we'll have to update the list with the correct version?

Reply:
Yea, I'll look into it some more. 

I'm pretty sure this tells us what functions/methods changed  in 8.1 though :).

Reply:
The php 8.0 installed in github actions on windows is returning reflection different from all the other tests environments I can find.

For example `phardata::compress` returns `null` for `getReturnType()` everywhere else except here.

Reply:
@orklah Updated the ignore arrays to support php versions like the original $ignoredFunctions does. This should work for php 8.0 - 8.2, but there are probably a couple missing from php 8.2.

Looks like your github actions config doesn't enable most of the extensions anymore, but I generated the lists with as as many extensions as I could load.

Reply:
Most of the methods not ignored for PHP 8.0 are missing reflection information. It doesn't exactly mean that CallMap is right or wrong.